src/main/java/io/github/eatmyvenom/litematicin/utils/Printer.java

```diff
@@ -373,7 +381,8 @@ public static ActionResult doPrinterAction(MinecraftClient mc) {
                                     return ActionResult.SUCCESS;
                                 }
                         	} else { // For survival
-                            	breaker.startBreakingBlock(pos, mc);
+				mc.interactionManager.attackBlock(pos, Direction.DOWN); //yes, this seemingly needless line adds functionality but paper would not allow it.
```

Comment:
If we attack the block, there is no need to also break it. Also i think it might be good to check for blocks that can be broken in one tick and add them to the if statement.
```suggestion
				if (!EASY_PLACE_MODE_PAPER.getBooleanValue()) {
				    mc.interactionManager.attackBlock(pos, Direction.DOWN); //yes, this seemingly needless line adds functionality but paper would not allow it.
				    return ActionResult.SUCCESS;
				}
```